### PR TITLE
Fix version injection

### PR DESCRIPTION
**Pull request by dryabinin94**

### CODE REVIEW

File: babel.config.js [hunk 1]
```diff
@@ -9,9 +9,7 @@ module.exports = (api) => {
       '@babel/plugin-proposal-optional-chaining',
       '@babel/plugin-proposal-nullish-coalescing-operator',
       // typescript supports class properties
-      '@babel/plugin-proposal-class-properties',
-      // inject __VERSION__ from package.json
-      'version-inline'
```
Comment (dryabinin-actionengine) [inline babel.config.js:14]: This line enable overwriting global `__VERSION__` variable from nearest (root)` package.json` file. But root `package.json` doesn't have `version` property therefore `__VERSION__` was undefined.
Now `__VERSION__` variable is setted in `test/browser.js` for browser tests and in `test/node.js` for node.js tests.

Comment (ibgreen) [reply babel.config.js:14]: 1. Isn't the root package.json still the nearest root in test folder?
2. The babel.config.js is used when transpiling each module. Removing it there probably means that the built library does not have a version.

File: modules/core/src/index.ts [hunk 2]
```diff
@@ -1,3 +1,4 @@
+import './lib/init';
```
Comment (ibgreen) [inline modules/core/src/index.ts:1]: This file wasn't used?

Comment (dryabinin-actionengine) [reply modules/core/src/index.ts:1]: It was used before. We using it for loaders global version injection.

File: babel.config.js (hunk 1 above)
Comment (dryabinin-actionengine) [reply babel.config.js:14]: 1. Root `package.json `is nearest for `babel.config.js `where 'version-inline' plugin was defined.
2. Yes you'r right. For transpiling each module we should use this plugin and it is already defined in `/scripts/webpack/bundle.js `and in `/scripts/webpack/worker.js `. I think that's enough.

**Review by ibgreen — APPROVED** [review]

**Comment by ibgreen** [comment]
The build system has changed.

**Pull request closed without merging**